### ISSUE TITLE
Feature Request: Add the option to keep Lock Key Notification showing on the screen till deselected.

### ISSUE THREAD

**Issue by chessset5** [ShankarBUS/ModernFlyouts#852]
**Is your feature request related to a problem? Please describe:**

I would like to be a way for the flyout to toggle in view when caps lock is enabled.

**Describe the solution you'd like:**

I would like to be a way for the flyout to toggle in view when caps lock is enabled.
Or some way to decide which lock keys toggle the flyout when enabled/pressed.

For instance I like to have my num lock always on, but I don't want the flyout to toggle on when num lock is on. I just want an indicator.
But for caps lock I want to know when it's activated on my screen with the flyout holding it's notification.

So basically I would like it if, audio, brightness, airplane mode, and all the individual lock keys had the ability to have set flyout times and flyout toggle options.

**Describe alternatives you've considered:**

I have increase the flyout time, but this isn't always a useful solution.

**Additional context:**

**Comment by a7ce** [comment]
Would appreciate it if you could change the Flyout Shortcut.

My keyboard doesn't have the Function key so I can't adjust the volume quickly.
